src/rust/ide/lib/parser/build.rs

```diff
@@ -25,7 +25,9 @@ use std::path::PathBuf;
 const PARSER_PATH: &str = "./pkg/scala-parser.js";
 
 /// Commit from `enso` repository that will be used to obtain parser from.
-const PARSER_COMMIT: &str = "890b800513a5b35376fc4f52beadb0dd869d3e70";
+/// Important note: Using commit not from Engine's master but from a backported branch
+///                 as the AST in this project hasn't been updated to conform to latest scala parser.
+const PARSER_COMMIT: &str = "6e0777535847adfe8d47a6f5196e7c0cdd32083c";
```

Comment:
What does it mean ? Can we / should we leave it this way @farmaazon ?

Reply:
To use newest parser from engine's main we should implement https://github.com/enso-org/ide/issues/732 which can be quite time-consuming.

I would only add here a issue number + `Engine's main` instead of `Engine's master`.